tasks/test_results_finisher.py

```diff
@@ -129,15 +131,15 @@ def process_impl_within_lock(
 
         escaper = StringEscaper(ESCAPE_FAILURE_MESSAGE_DEFN)
 
+        failures = []
```

Comment:
maybe also remove the definition of `failures` in line 130? It seems to be replaced by this one.